utils.go

```diff
@@ -277,6 +277,9 @@ func (s *Storage) formatFileObject(v *gs.ObjectAttrs) (o *typ.Object, err error)
 	if value := v.StorageClass; value != "" {
 		sm[MetadataStorageClass] = value
 	}
+	if value := v.CustomerKeySHA256; value != "" {
+		sm[MetadataSseCustomerKeySha256] = value
```

Comment:
How about keeping the naming style with SDK?

- `sse_customer_key` => `customer_key`
- `sse_customer_key_sha256` => `customer_key_sha256`

Or the naming in GCS's docs:

- `encryption_key`
- `encryption_key_sha256`

Reply:
I wrote a RFC about this: https://github.com/aos-dev/specs/pull/38